code/modules/power/engines/singularity/particle_accelerator/particle_control.dm

```diff
@@ -247,36 +213,42 @@
 			part.update_icon()
 	return 1
 
+/obj/machinery/particle_accelerator/control_box/ui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = FALSE, datum/tgui/master_ui = null, datum/ui_state/state = GLOB.default_state)
+	ui = SStgui.try_update_ui(user, src, ui_key, ui, force_open)
+	if(!ui)
+		ui = new(user, src, ui_key, "ParticleAccelerator", name, 350, 160, master_ui, state)
+		ui.open()
+
+/obj/machinery/particle_accelerator/control_box/ui_data(mob/user)
+	var/list/data = list()
+	data["assembled"] = assembled
+	data["power"] = active
+	data["strength"] = strength
+	return data
+
+/obj/machinery/particle_accelerator/control_box/ui_act(action, list/params, datum/tgui/ui, datum/ui_state/state)
+	if(..())
+		return
 
```

Comment:
I think you should readd the variable check for `interface_control` here, that was in topic with the `to_chat(usr, "<span class='error'>ERROR: Request timed out. Check wire contacts.</span>")` line

Reply:
Oops. 
Resolved with [b777308](https://github.com/ParadiseSS13/Paradise/pull/21370/commits/b7773087a15a721ce43a60e6fe12af319006b2b3)